### PR TITLE
Feature site mix component

### PR DESCRIPTION
**Pull request by skittan4**

### CODE REVIEW

File: src/components/SiteMix/SiteMix.jsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      const actData = await response.json()
+      let actual = []
+      let actualLabels = []
+      actData.data.forEach((element) => {
+        actualLabels.push(element.cost_level)
+        actual.push(parseInt(element.site_ee))
+      })
+
+      const plannedresponse = await fetch(
+        'http://localhost:4000/GetPlannedSiteMix/2',
```
Comment (tawfiknasser) [inline src/components/SiteMix/SiteMix.jsx]: can you use config here ?

File: src/components/SiteMix/SiteMix.jsx [hunk 2]
```diff
@@ -22,7 +25,7 @@ export default function SiteMix() {
       })
 
       const plannedresponse = await fetch(
-        'http://localhost:4000/GetPlannedSiteMix/2',
+        `${process.env.REACT_APP_API_URL}/GetPlannedSiteMix/2`,
```
Comment (tawfiknasser) [inline src/components/SiteMix/SiteMix.jsx]: https://github.com/WebAhead-cs2/caps-look/blob/development/src/config.js

This config

File: src/components/SiteMix/SiteMix.jsx (hunk 1 above)
Comment (skittan4) [reply src/components/SiteMix/SiteMix.jsx]: fixed in SiteMix.jsx and Login .jsx

File: src/components/SiteMix/SiteMix.jsx (hunk 2 above)
Comment (tawfiknasser) [reply src/components/SiteMix/SiteMix.jsx]: Sorry if I wasn't clear

Comment (tawfiknasser) [reply src/components/SiteMix/SiteMix.jsx]: @skittan4

Comment (skittan4) [reply src/components/SiteMix/SiteMix.jsx]: resolved

**Review by tawfiknasser — APPROVED** [review]

**Pull request merged**